renderer.php

```diff
@@ -114,34 +114,34 @@ protected function view_page_content($poster) {
         $cssclassmain .= $haspre ? '' : ' empty-region-mod_poster-pre';
         $cssclassmain .= $haspost ? '' : ' empty-region-mod_poster-post';
 
-        $out .= html_writer::start_div($cssclassmain, array('id' => 'mod_poster-content'));
+        $out .= \html_writer::start_div($cssclassmain, array('id' => 'mod_poster-content'));
```

Comment:
Why is this change needed? I prefer not to have these backslashes in non-namespaced files personally.